src/pipecat/services/ai_services.py

```diff
@@ -137,6 +141,13 @@ async def process_generator(self, generator: AsyncGenerator[Frame | None, None])
                     await self.push_frame(f)
 
 
+@dataclass
+class FunctionCallEntry:
```

Comment:
nit: `FunctionEntry` might be better because it pertains to a registered function, not an invocation of a function

Reply:
You are right.